psake.ps1

```diff
@@ -17,48 +20,151 @@ task Init {
     "Build System Details:"
     Get-Item ENV:BH*
 
-    $modules = 'Pester', 'platyPS', 'PSScriptAnalyzer'
-    Install-Module $modules -Repository PSGallery -Confirm:$false
-    Import-Module $modules -Verbose:$false -Force
+    'Pester', 'platyPS', 'PSScriptAnalyzer' | Foreach-Object {
+        if (-not (Get-Module -Name $_ -ListAvailable -ErrorAction SilentlyContinue)) {
+            Install-Module $modules -Repository PSGallery -Scope CurrentUser -Confirm:$false -ErrorAction Stop
```

Comment:
Should this line be:

```powershell
Install-Module $_ -Repository PSGallery -Scope CurrentUser -Confirm:$false -ErrorAction Stop
```

Looking in VS Code, I can't find the definition for $modules, assume it existed earlier.